Mathlib/Topology/Algebra/ContinuousMonoidHom.lean

```diff
@@ -286,47 +286,24 @@ theorem embedding_toContinuousMap :
 #align continuous_monoid_hom.embedding_to_continuous_map ContinuousMonoidHom.embedding_toContinuousMap
 #align continuous_add_monoid_hom.embedding_to_continuous_map ContinuousAddMonoidHom.embedding_toContinuousMap
 
+@[to_additive]
+lemma range_toContinuousMap :
+    Set.range (toContinuousMap : ContinuousMonoidHom A B → C(A, B)) =
+      {f : C(A, B) | f 1 = 1 ∧ ∀ x y, f (x * y) = f x * f y} := by
+  refine Set.Subset.antisymm (Set.range_subset_iff.2 fun f ↦ ⟨map_one f, map_mul f⟩) ?_
+  rintro f ⟨h1, hmul⟩
+  exact ⟨{ f with map_one' := h1, map_mul' := hmul }, rfl⟩
```

Comment:
```suggestion
      {f : C(A, B) | f 1 = 1 ∧ ∀ x y, f (x * y) = f x * f y} :=
  (Set.range_subset_iff.2 fun f ↦ ⟨map_one f, map_mul f⟩).antisymm
    fun f ⟨h1, hmul⟩ ↦ ⟨{ f with map_one' := h1, map_mul' := hmul }, rfl⟩
```
(untested)

Reply:
Doesn't work because this way Lean doesn't know the type of `⟨map_one f, map_mul f⟩`.